StepDefinitions/HotelsSD/HotelAddGusSD.js

```diff
@@ -0,0 +1,53 @@
+// const { Then, Given } = require("@wdio/cucumber-framework");
+// const { expect } = require("chai");
+// const HotelHomePage = require("../../Pages/Hotels/HotelHomePage");
+
+// const hotelhomepage = new HotelHomePage();
+
+// Given(/^I am on hotels homepage$/, async function() {
+//     await browser.url('/');
+//     await browser.pause(1000);
+// })
+
+// Then(/^I click on Travelers$/, async function() {
+//     await hotelhomepage.clickTravelersBox();
+//     await browser.pause(2000);
```

Comment:
This hard coded pause() isn't professional and not efficient as well. Instead of waiting after the action you can wait for element to be displayed/enabled before performing the action. eg. you can do clickTravelersBox() like below

async clickTravelersBox() {
    const traverlsBox = await $(this.travelersboxLocator);
     await traverlsBox.waitForDisplayed(); // or waitForEnabled();
    await traverlsBox.click();
}

Try to follow this for all functions, then it'll look more professional.